web/src/routes/OrganizationsList/Organization/Tabs/Settings/BillingInformation.tsx

```diff
@@ -0,0 +1,454 @@
+            {currentPlan?.plan.toUpperCase()} organization&apos;s plan
+          </Title>
+        </FlexItem>
+        {privateAllowed ? (
+          <FlexItem>{`${privateCount} of ${privateAllowed} private repositories used`}</FlexItem>
+        ) : null}
+        <FlexItem>{`${totalResults} of unlimited public repositories used`}</FlexItem>
+      </Flex>
+
+      <Flex width={'70%'}>
+        <Button
+          variant="secondary"
+          isDisabled={true} // TODO: Enable when we have a payment processor
+          onClick={async () => {
+            await upgrade();
+          }}
+        >
+          Change Plan
+        </Button>
+      </Flex>
+
+      <FormGroup fieldId="checkbox">
+        <Checkbox
+          id="checkbox"
+          label="Send receipts via email."
+          aria-label="Send receipts via email"
+          isChecked={invoiceEmail}
+          onChange={() => {
+            setTouched(true);
+            setInvoiceEmail(!invoiceEmail);
+          }}
+        />
+      </FormGroup>
+      <FormGroup
+        isInline
+        label="Invoice Email"
+        fieldId="form-email"
+        validated={validated}
+        helperTextInvalid="Must be an email"
+        helperTextInvalidIcon={<ExclamationCircleIcon />}
+        helperText="Invoices will be sent to this e-mail address."
+      >
+        <TextInput
+          type="email"
+          id="billing-settings-invoice-email"
+          validated={validated}
+          value={invoiceEmailAddress}
+          onChange={(val) => {
+            setTouched(true);
+            setInvoiceEmailAddress(val);
+            if (/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(val)) {
+              setValidated('success');
+            } else if (!val) {
+              setValidated('default');
+            } else {
+              setValidated('error');
+            }
+          }}
+        />
+      </FormGroup>
+
+      <Button
+        variant="link"
+        isInline
+        isDisabled={true} // TODO: Enable when we have a payment processor
+      >
+        View Invoices
+      </Button>
+
+      {isUserOrganization && (
+        <>
+          <Title headingLevel="h3">Account Type</Title>
+          <Radio
+            id="radio-individual"
+            label="Individual Account"
+            name="radio-individual"
+            isChecked={accountType == 'individual'}
+            onClick={() => setAccountType('individual')}
+            description="Single account with multiple repositories"
+          />
+          <Radio
+            id="radio-organization"
+            label="Organization Account"
+            name="radio-organization"
+            onClick={() => setAccountType('organization')}
+            isChecked={accountType == 'organization'}
+            description="Multiple users and teams that share access and billing under a single namespace"
+          />
+          {user.organizations.length > 0 && accountType == 'organization' && (
```

Comment:
The view details still appears for a user that doesn't belong to any organizations.